support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -429,7 +496,28 @@ function WeeklyCheckoutForm(props: PropTypes) {
 						price={props.discountedPrice.price}
 						currency={props.currencyId}
 					/>
-					<PaymentTerms paymentMethod={props.paymentMethod} />
+					{inThreeTierTestVariant ? (
+						<ThreeTierTerms
+							paymentMethod={props.paymentMethod}
+							total={standardDigitalPlusPrintPrice}
+							currencySymbol={currencies[props.price.currency].glyph}
+							paymentFrequency={billingPeriodNounLowerCase}
+							discount={
+								digitalPlusPrintPotentialDiscount
```

Comment:
Used here and on line 238 - maybe worth putting in a seperate function `getDiscount`?